src/styles/product-card.css

```
@@ -0,0 +1,117 @@
+/* #region product-card */
+.product-card {
+  width: 200px;
+  height: 406px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  margin: 0 3px 36px;
+}
+
+.product-card__title {
+  margin-bottom: 4px;
+}
+
+.product-card__card-id {
+  margin-bottom: 16px;
+}
+
+.product-card__rate {
+  margin-bottom: 20px;
+}
+
+.product-card__price {
+  margin-bottom: 16px;
+}
+
+.product-card__container {
+  margin: 32px 16px 16px;
+}
+/* #endregion */
+
+/* #region image */
+.product-image {
+  width: 100%;
+  height: 134px;
+}
+/* #endregion */
+
+/* #region title, id, rate, price */
+.title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  color: #060b35;
+}
+
+.card-id {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  color: #616070;
+}
+
+.rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.rate-review {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+}
+
+.price-name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+
+  color: #060b35;
+}
+/* #endregion */
+
+/* #region button */
+.product-card-button {
```

```suggestion
.buy-bottun {
```

якщо ти робиш кнопку блоком то не потрібно її привязувати картки. Це просто кнопка для покупки